Update on "Page Load" component (reviewed) (#13333)
cloudflare/cloudflare-docs: content/analytics/web-analytics/understanding-web-analytics/page-load-time-summary.md

Below is a list of all the components you can inspect:

> - **Page load**: The total amount of time required to load the page.(Note: The Page load time does not match the sum of the other timings available in Web Analytics. This is because the page load time also includes timings that are not shown. These include Pre-DNS-lookup timings and unattributed gaps between timing metrics.)

Reviewer: ```suggestion
- **Page load**: The total amount of time required to load the page. Note that page load time does not correspond to to the sum of the other timings available in Web Analytics. This happens because the page load time also includes timings that are not displayed, such as pre-DNS lookup timings and unattributed gaps between timing metrics.
```
